### DISCUSSION TITLE
Clickable label spans

### DISCUSSION THREAD

**Discussion by IeuanWalker** [CommunityToolkit/Maui#191] [New Feature Discussions]
Moving an enhancement from XCT to MCT.

Two issues was raised for this feature and a PR was created which implemented both features and allowed for custom implementations.

Issues - 
- https://github.com/xamarin/XamarinCommunityToolkit/issues/171
- https://github.com/xamarin/XamarinCommunityToolkit/issues/178

PR - 
- https://github.com/xamarin/XamarinCommunityToolkit/pull/172

PR was originally closed because of an issue in Xamarin.Forms where multiple tap gestures triggering on android, see https://github.com/xamarin/Xamarin.Forms/issues/11650. But that should be fixed now _(haven't had time to test it myself)_.

If it is fixed the only thing left from the implementation is to move the command to the individual match types, see this comment - https://github.com/xamarin/XamarinCommunityToolkit/pull/172#issuecomment-686717861

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
